kernel-build-util.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# UTILITY SCRIPT FOR BUILDING KERNEL AND RUNNING IT WITH QEMU
```

Comment:
I'll let this script in for now, but later I might try to add this functionality to the makefile